src/app/state/rest/index.ts

```diff
@@ -98,6 +99,16 @@ export const useBAGWithZipCode = (bagId: string, options?: Options) => {
   })
 }
 
+export const useBAGLodging = (type: string | undefined, subTypeId: string | undefined, options?: Options) => {
+  const handleError = useErrorHandler()
+  return useApiRequest<BAGObjectResponse>({
+    url: `https://api.data.amsterdam.nl/bag/v1.1/${ type }/${ subTypeId }/`,
+    ...options,
+    groupName: "dataPunt",
+    handleError
```

Comment:
I would add lazy here, instead of in `src/app/features/addresses/components/atoms/ObjectDetails/ObjectDetails.tsx`. E.g. `lazy: type === undefined || subTypeId === undefined`.

Reply:
Done